### PR TITLE
LPS-161837 SF

### PR DESCRIPTION
**Pull request by Claudionilton**

### CODE REVIEW

**Comment by Claudionilton** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
Just starting reviewing :)

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/NewQuestion.es.js [hunk 1]
```diff
@@ -200,6 +198,8 @@ export default withRouter(
 									value={headline}
 								/>
 
+								<span className="d-flex justify-content-end mt-2 text-secondary">{`${headline.length} / 75`}</span>
```
Comment (kevenleone) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/NewQuestion.es.js:203]: **75** is a magic number, and shouldn't be placed here.

You can create a `const` for example `const HEADLINE_MAX_LENGTH = 75` and reuse it where **75** is placed, the same for the <input in this file

The const can be placed before `withRouter`

**Review by kevenleone — CHANGES_REQUESTED** [review]

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/NewQuestion.es.js (hunk 1 above)
Comment (kevenleone) [reply modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/NewQuestion.es.js:203]: image

Also limit count is alone in one line, and the description in another. Would be possible to place both texts in the same line?

Comment (kevenleone) [reply modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/NewQuestion.es.js:203]: @Claudionilton

**Comment by Claudionilton** [comment]
ci:test:sf

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/NewQuestion.es.js [hunk 2]
```diff
@@ -37,6 +37,7 @@ import {
 	slugToText,
 	useDebounceCallback,
 } from '../../utils/utils.es';
+const HEADLINE_MAX_LENGTH = 75;
```
Comment (kevenleone) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/NewQuestion.es.js:40]: @Claudionilton Add an empty line before this one

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**